### DISCUSSION TITLE
Fanuc

### DISCUSSION THREAD

**Discussion by petrasvestartas** [visose/Robots#160] [Q&A] [answered]
Is there any possibility to control fanuc using Robots?

**Comment by visose** [comment, marked as answer]
https://github.com/visose/Robots/blob/c3a58673acc1ee5b548aab3be7d62677e34db736/src/Robots/PostProcessors/FanucPostProcessor.cs#L15

Not yet. It might happen if: I get to work on a project that uses Fanuc, a contributor adds this, or I get contracted/sponsored to implement Fanuc in Robots (the support of Staubli was sponsored).

**Reply by petrasvestartas** [reply]
Thank you for the answer;)

**Comment by petrasvestartas** [comment]
@visose 

Do you know why fanuc are so not popular in universities?
Do they have more disadvantages than abb or kuka?

**Reply by visose** [reply]
I have no idea, I never used them, but my guess is they are similar to other industrial robots.

In general, people choose specific products because of familiarity, existing ecosystem, ability to hire people that are familiar with it, far along might be price, and at the end of the list are any technical advantages, unless they are groundbreaking, and sometimes not even so. 

With robots, there's also some geographical variation on where different manufacturers invest in sales and support.
